symfony/src/Entity/Site.php

```diff
@@ -276,4 +276,9 @@ public function setDeleted(int $deleted): self
 
         return $this;
     }
+
+    public function toArray()
+    {
+        return get_object_vars($this);
+    }
```

Comment:
While in the current case, this works fine, I think it would be a better practice to not automatically expose all of the object variables like this. We should consider using the [Serializer component](https://symfony.com/doc/4.4/components/serializer.html). (In the current case where this is used for the site sync diff, you would only need to normalize, and not serialize).